src/domain/entities/NoteList.ts

```diff
@@ -0,0 +1,9 @@
+import type { Note } from './Note';
+
+/**
+ * Note list  entity.
+ * An object with the "items" property containing a list of all existing notes created by the user
+ */
+export type NoteList = {
+    items: Note[];
```

Comment:
seems like there is 4 space indentation instead of 2. Install "editorconfig" plugin to your IDE so it will respect the `.editorconfig` file